### PR TITLE
Color branding & Two way video flavor

### PR DESCRIPTION
**Pull request by evanisnor**
* Branding overrides included in the advanced flavour
* Updating permissions in the manifest - they were not quite correct and causing problems on M devices
* Two-way video flavour

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -33,15 +33,21 @@ android {
...
 
         // The advanced flavor demonstrates more involved customization including replacing portions
-        // of logic for a more tightly integrated experience.
+        // of logic and color branding for a more tightly integrated experience.
         advanced {
 
         }
 
+        // The two-way video flavour allows the user to transition from a screen sharing session
+        // to a video call session by exposing a Camera button on the
```
Comment (mattcreaser) [inline app/build.gradle]: incomplete comment

File: app/src/advanced/res/values/sos_overrides.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<resources>
```
Comment (mattcreaser) [inline app/src/advanced/res/values/sos_overrides.xml]: Do you think this makes sense to go in `basic`?

File: app/build.gradle (hunk 1 above)
Comment (mattcreaser) [inline app/build.gradle]: This could probably just be added to the advanced flavor right?

Comment (evanisnor) [inline app/build.gradle]: I figured we might want to demonstrate two-way video as something that is added to the Halo UI's default configuration. I think doing it this way makes it very clear how the two features fit together.

File: app/src/advanced/res/values/sos_overrides.xml (hunk 2 above)
Comment (evanisnor) [inline app/src/advanced/res/values/sos_overrides.xml]: I made the assumption that `basic` meant "out of the box experience", which would include the default palette. Advanced was the next logical place to override the colours.

Comment (mattcreaser) [inline app/src/advanced/res/values/sos_overrides.xml]: Simple is "out of the box", see the `README` for a description of the flavors.

File: app/build.gradle (hunk 1 above)
Comment (mattcreaser) [inline app/build.gradle]: Ok, you'll need to update the `README`.

**Comment by mattcreaser** [comment]
LGTM

**Pull request merged**